### DISCUSSION TITLE
Train voices with better quality. What are the limits?

### DISCUSSION THREAD

**Discussion by kvrban** [rhasspy/piper#539] [General]
A basic question, where are the limits in piper itself as far as the best possible quality of a voice is concerned?

Or to put it another way, can piper be used to train voices that are on a par with ElevenLabs? assuming that you would produce high-quality training material? So recorded by a sound engineer in the studio and many hours of material.

**Comment by rmcpantoja** [comment]
In theory, it is possible.
But, you need to train a model from scratch, there aren't pretrained models in 22.05k avobe.
To train, for example, a 44.1k voice, you'll need to adjust some parametters during preprocessing. I did this in my ISTFT implementation, see: https://github.com/rmcpantoja/piper/blob/MB-ISTFT-VITS2/src/python/piper_train/__main__.py

**Reply by zaidifm** [reply]
Hi! I’m very curious about this as I’ve been thinking about training an MB-ISTFT-VITS2 model. 

Do I understand correctly that your branch integrates MB-ISTFT-VITS2 models into the Piper framework? What differences are there between training / inferencing such a model through the Piper framework, rather than using the  MB-ISTFT-VITS2 repo?
